src/components/grid/col.tsx

```diff
@@ -4,35 +4,28 @@ import * as React from 'react';
 import { Col as StyledCol } from './style';
 
 export interface ColProps extends React.HTMLAttributes<HTMLDivElement> {
-  xs?: number | string;
-  s?: number | string;
-  m?: number | string;
-  ml?: number | string;
-  l?: number | string;
-  xl?: number | string;
+  xs?: number | string | null;
```

Comment:
maybe better to define a separate type to not repeat yourself?

type ColSize = number | string | null